en/docs/develop-components/develop-services/develop-a-grpc-service.md

```diff
@@ -0,0 +1,210 @@
+
+9. Click **Create**. Once the component creation is complete, you will see the component's overview page.
+
+You have successfully created a Service component from a Dockerfile. Now let's build and deploy the service.
+
+## Step 2: Configure the service port with endpoints
+
+We expect to run our gRPC server Service component on port 8080. To securely expose the service through Choreo, we must provide the port and other required information to Choreo. In Choreo, we expose our services with endpoints. You can read more about endpoints in our [endpoint documentation](https://wso2.com/choreo/docs/develop-components/develop-services/develop-a-service/#what-are-endpoints-in-service-components).
+
+Choreo looks for an endpoints.yaml file inside the `.choreo` directory to configure the endpoint details of a containerized component. Place the `.choreo` directory at the root of the Docker build context path.
+
+In our gRPC server sample, the endpoints.yaml file is at `go/grpc/.choreo/endpoints.yaml`. Our build context path is `go/grpc`.
+
+## Step 3: Build and deploy
+
+Having connected the source repository and configured the endpoint details, we can now proceed to build and deploy the gRPC server Service component.
+
+To build and deploy the service, follow these steps:
+
+1. On the Deploy page, click **Deploy Manually**.
+
+    !!! note
+        Deploying the service component may take a while. You can track the progress by observing the logs. Once the deployment is complete, the deployment status changes to **Active** in the corresponding environment card.
+
+
+2. Check the deployment progress by observing the console logs on the right of the page.
+    You can access the following scans under **Build**. 
+
+    - **The Dockerfile scan**: Choreo performs a scan to check if a non-root user ID is assigned to the Docker container to ensure security. If no non-root user is specified, the build will fail.
+    - **Container (Trivy) vulnerability scan**: This detects vulnerabilities in the final docker image. 
+    -  **Container (Trivy) vulnerability scan**: The details of the vulnerabilities open in a separate pane. If this scan detects critical vulnerabilities, the build will fail.
+
+!!! info
+    If you have Choreo environments on a private data plane, you can ignore these vulnerabilities and proceed with the deployment.
+
+3. On the **Configure & Deploy** pane, click **Deploy**.
+4. Once you have successfully deployed your service,  navigate to the component overview page and copy the gRPC service URL. You need to provide that URL when setting up the client application later in this guide.
+
+Now we have successfully deployed the gRPC server. Currently, the gRPC service services are only accessible for the components deployed within the same project.
```

Comment:
Better to avoid the term `we`.

Reply:
Need to remove the repetitive service term.

Reply:
Done